.circleci/config.yml

```diff
@@ -62,6 +62,33 @@ jobs:
       - run: docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
       - run: make docker-push
 
+  publish_release:
+    docker:
+      # Available from https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/improbable-eng/thanos
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
+      - run: make promu
+      - run: promu crossbuild tarballs
+      - run: promu checksum .tarballs
+      - run: promu release .tarballs
+      - store_artifacts:
+          path: .tarballs
+          destination: releases
+      - run: ln -s .build/linux-amd64/thanos thanos
+      - run: make docker DOCKER_IMAGE_TAG=$CIRCLE_TAG
```

Comment:
I don't get this. `make docker` does not use `DOCKER_IMAGE_TAG` param

Reply:
Ahh, we do use it in Prometheus with `docker build -t "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .`

Reply:
yea, let's kill the DOCKER_IMAGE_TAG thing then, and just push `vX.X.X`